static/js/browse.js

```diff
@@ -3,7 +3,7 @@ window.sessionStorage.setItem("TvTrackerUsername", "testUser");
 function saveMediaEntry(mediaEntry) {
     const username = window.sessionStorage.getItem("TvTrackerUsername");
     const token = window.sessionStorage.getItem("TvTrackerToken");
-    const URL = "/addMediaEntry?username=" + username + "&token=" + token
+    const URL = "/addMediaEntry?username=" + username + "&token=" + token;
```

Comment:
The semi colons were auto added by [SolarLint](https://www.sonarlint.org/intellij). I added the SolarLint plugin to the project. SolarLint is a code linting tool which enables us to adhere to many of the best practices found here: https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3  

In Intellji, click on Analyze -> Inspect Code to generate a full report, or Cleanup Code to allow the tool to clean up errors.

Reply:
SolarLint... very neat!